kqueen/blueprints/api/generic_views.py

```diff
@@ -278,6 +278,11 @@ def dispatch_request(self, *args, **kwargs):
             try:
                 self.save_object()
                 self.after_save()
+            except ValueError as e:
+                if 'unique' in str(e):
```

Comment:
IMO its bad case to manage error codes by string-checking, traceback can change when package updates
and IMO both this cases can be managed by 400 code

Reply:
ok, I'll think how it will be better to handle that